app/templates/signup.html

```diff
@@ -2,18 +2,25 @@
 
 {% block content %}
 
-<h1>Sign Up & Register</h1>
+<html>
+ <div class="myDiv">
+<link rel="stylesheet" href = "{{ url_for('static', filename='styles/newacct.css') }}">
+<h1>Sign Up </h1>
+
     <form action = "" method = "post" novalidate>
 
         {{ form.hidden_tag() }}
 
+         <p>Please fill in this form to create an account.</p>
+        <div class="myDiv">
         <p>
             {{ form.email.label }}<br>
             {{ form.email(size = 32) }}
             {% for error in form.email.errors %}
             <span style="color: red;">[{{ error }}]</span>
```

Comment:
nit: avoid inline styles since they can be harder to manage

Reply:
Not sure what you mean here

Reply:
the paragraph tag?

Reply:
@Kamoa17 I was referencing this line `<span style="color: red;">[{{ error }}]</span>`

Reply:
okay im just going to remove it from the HTML, is there any other issues with the pr